steam-tweaks

```diff
@@ -1,5 +1,6 @@
 #! /bin/bash
 
+migrate-to-chimera --logout
```

Comment:
I don't think we want to logout here. Especially for ChimeraOS.

Reply:
Yeah, I think we do want to logout, this will run before steam and tweaks, but will not certainly run before chimera and probably will mess with settings in an unpredictable way. Loging out the user guarantees that:
- all user services are stopped and we start with a clean session.
- lightdm will restart the session automatically and we start anew.
and since this is done before any graphics are shown, then it will be seamless for the user. (aside for a "long waiting time at a black screen")

Reply:
Oh right, I forgot about chimera, but I am not sure that will actually restart the chimera session?

Reply:
The way we have set it up yes. Systemd will respawn LightDM and since we configured the seat to be auto login with user `gamer` then it will login again. Is the same behavior you get when you exit/kill steam in our session.

Reply:
Yes, but I was pretty sure that chimera continues running after you log out. At least I recall that was the case when I was testing something. I'll test that again.

Reply:
That was true when `chimera.service` was a system service and not a user one. We are running it on the user scope for a while now.

Reply:
Just tested logging out the user on my box and the session restarted.

Reply:
Just tested logging out the user on my box and the session restarted on its own along with chimera app